declarations/validator.d.ts

```diff
@@ -151,6 +151,7 @@ declare module 'validator' {
   export function isTaxID(str: string, locale: import('../src/options').TaxIDLocale): boolean;
   export function isTime(str: string, options?: import('../src/options').IsTimeOptions): boolean;
   export function isURL(str: string, options?: import('../src/options').IsURLOptions): boolean;
+  export function isULID(str: string): boolean;
```

Comment:
Hey, this should only be for validator.js types. It seems it doesn't have a `isULID` yet.

Did you mean to wait for https://github.com/validatorjs/validator.js/pull/2294 to be merged?

If you rather have it in express-validator ASAP, I think you can remove it from here and manually document it in https://github.com/express-validator/express-validator/blob/master/docs/api/validation-chain.md.

Reply:
@gustavohenke Thanks for the review.
This PR has no dependency on https://github.com/validatorjs/validator.js/pull/2294 so i'm removing the lines according to your suggestion.